specs/gltfExtensions/KhrLightsPunctualValidationSpec.ts

```diff
@@ -0,0 +1,18 @@
+import { Validators } from "../../src/validation/Validators";
+
+describe("KHR_lights_punctual extension validation", function () {
+  it("detects no issues in validTilesetWithKhrLightsPunctual when no temporal traits are used", async function () {
+    const result = await Validators.validateTilesetFile(
```

Comment:
While it might not be strictly necessary to validate a _tileset_ file that contains the valid glTF, I think that this makes sense here. I makes sure that the validator does properly handle a tileset with `KHR_lights_punctual` content (and does not generate issues/warnings for that). Particularly because the `MaxarTemporalLightTraitsValidator` is "nested" in the `KhrLightsPunctualValidator`. So this establishes the baseline on which the `MaxarTemporalLightTraitsValidationSpec` builds.

But also see the note about the `extensionsUsed` validation below, and in the final/summarizing review comment!